src/main/java/uk/gov/pay/publicauth/resources/PublicAuthResource.java

```diff
@@ -110,10 +112,13 @@ public Response updateTokenDescription(JsonNode payload) {
             if (updated) {
                 Optional<Map<String, Object>> tokenData = authDao.findTokenByTokenLink(tokenLink);
                 return tokenData
-                        .map(token -> ok(token).build())
+                        .map(token ->  {
+                            LOGGER.info("Updated description of token with token_link {}", tokenLink);
+                            return ok(token).build();
+                        })
                         .orElseGet(() -> serverErrorResponse(LOGGER, "An exception occurred while finding the updated token with link: " + tokenLink));
             }
-            return notFoundResponse(LOGGER, "Could not update token description");
+            return notFoundResponse(LOGGER, "Could not update description of token with token_link " + tokenLink);
```

Comment:
- I know this was before but passing a logger as parameter is not a good practice.
- Can we create an story to refactor it?

Reply:
Sure :+1: doing it right now